### PR TITLE
Updating migration guides

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Run_results should have been in v1.7 - moving the migration guide items and updating version block
Adding items new to 1.8 to migration guide.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 1]
```diff
@@ -60,21 +60,30 @@ dbt test --select "test_type:data"           # run all data tests
...
 
-The run_results.json now includes three attributes related to the `applied` state that complement `unique_id`:
+```
 
-- `compiled`: Boolean entry of the node compilation status (`False` after parsing, but `True` after compiling).
-- `compiled_code`: Rendered string of the code that was compiled (empty after parsing, but full string after compiling).
-- `relation_name`: The fully-qualified name of the object that was (or will be) created/updated within the database.
+### The `--empty` flag
 
+The [`run`](/reference/commands/run#the-`--empty`-flag) and [`build`](/reference/commands/build#the---empty-flag) commands now support the `--empty` flag for building schema-only dry runs. The `--empty` flag limits the refs and sources to zero rows. dbt will still execute the model SQL against the target data warehouse but will avoid expensive reads of input data. This validates dependencies and ensures your models will build properly.
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md:82]: How do you know the dependencies have been validated?

**Review by runleonarun — COMMENTED** [review]
Hey @matthewshaver Just have one question about the --empty flag

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md:82]: If the models build successfully. It's not much different than a normal `build` or `run` command except it limits the rows to zero.

Comment (runleonarun) [reply website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md:82]: OK gotcha! So they see passing tests. That makes sense. Carry on...nothing to see here...I will approve

**Review by runleonarun — APPROVED** [review]
:shipit:

**Pull request merged**